src/client/firefox/fronts-device.js

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+export function getDeviceFront() {
+  return {
+    getDescription: function() {
+      return {
+        // Return anything that will not match Fennec v59
```

Comment:
Maybe this should say Fennec 60, now that you've changed the rest to mention 60 as well?